server/internal/usecase/interactor/request.go

```diff
@@ -58,7 +58,7 @@ func (r Request) Create(ctx context.Context, param interfaces.CreateRequestParam
 			return nil, err
 		}
 
-		repoItems, err := r.repos.Item.FindByIDs(ctx, param.Items.IDs())
+		repoItems, err := r.repos.Item.FindByIDs(ctx, param.Items.IDs(), nil)
```

Comment:
it should be updated similar to Update method. 

1. Find public versions
2. Check if contains latest ref: `item.Refs().Has(version.Latest)`
3. If so, return an error